### PR TITLE
Added apoc-core

### PR DESCRIPTION
**Pull request by renetapopova**
Trello cards:
* [Add apoc-core as a plugin to docker](https://trello.com/c/ezWQQW1z/536-add-apoc-core-as-a-plugin-to-docker)
* [Update docs about using `apoc core` with docker](https://trello.com/c/UQ2MhGpI/2926-update-docs-about-using-apoc-core-with-docker)

### CODE REVIEW

File: docker-image-src/4.4/docker-entrypoint.sh [hunk 1]
```diff
@@ -213,7 +213,12 @@ function install_neo4j_labs_plugins
   local _old_config="$(mktemp)"
   cp "${NEO4J_HOME}"/conf/neo4j.conf "${_old_config}"
   for plugin_name in $(echo "${NEO4JLABS_PLUGINS}" | jq --raw-output '.[]'); do
-    load_plugin_from_github "${plugin_name}"
+    local _location="$(jq --raw-output "with_entries( select(.key==\"${_plugin_name}\") ) | to_entries[] | .value.location" /neo4jlabs-plugins.json )"
+    if "${_location}" != "null"; then
+        install_plugin_from_location "${_location}"
```
Comment (eastlondoner) [inline docker-image-src/4.4/docker-entrypoint.sh:218]: @renetapopova you'll need to implement install_plugin_from_location ;-)

**Review by eastlondoner — CHANGES_REQUESTED** [review]

**Comment by eastlondoner** [comment]
Nice try.

Install_plugin_from_location
needs to take a local location (/var/lib/neo4j/labs/*) and copy it to the plugins directory using cp

File: neo4jlabs-plugins.json [hunk 2]
```diff
@@ -5,14 +5,20 @@
       "dbms.security.procedures.unrestricted": "apoc.*"
     }
   },
+  "apoc-core": {
+    "location": "/labs/apoc-core-*.jar",
```
Comment (eastlondoner) [inline neo4jlabs-plugins.json:9]: I think this should be a full path so /var/lib/neo4j/labs...

File: docker-image-src/4.4/docker-entrypoint.sh [hunk 3]
```diff
@@ -146,6 +146,30 @@ function check_mounted_folder_writable_with_chown
     fi
 }
 
+function install_plugin_from_location
+{
+  # Install a plugin from location at runtime.
+  local _plugin_name="${1}" #e.g. apoc-core
```
Comment (eastlondoner) [inline docker-image-src/4.4/docker-entrypoint.sh:152]: The passed argument will be the location not plug-in name

**Review by eastlondoner — CHANGES_REQUESTED** [review]

**Comment by renetapopova** [comment]
@eastlondoner I decided to keep the `plugin_name`as a parameter so that the function follows the same pattern as the rest; I can validate that the destination is readable, and the message is the same. I haven't tested it, I am also not sure what will happen if there are more apoc-core jar files in the location (in my Neo4j's` /labs` folder there are no jars), I guess the function will copy them all one by one.

**Review by eastlondoner — COMMENTED** [review]
@renetapopova this looks great - leave the PR up and I will add a test

**Review by jennyowen — APPROVED** [review]

**Review by eastlondoner — APPROVED** [review]
LGTM please Squash and merge :-)

**Pull request merged**